ansible/roles/test/tasks/advanced-reboot.yml

```diff
@@ -20,6 +20,21 @@
     - fail: msg="Please set vm_hosts variable with a list of VMs"
       when: vm_hosts is not defined
 
+    - name: Preboot-list initialization
+      set_fact: preboot_list={% if preboot_list is not defined %}[None]{% else %}{{ preboot_list }}{% endif %}
+
+    - name: Preboot files initialization
+      set_fact: preboot_files={% if preboot_files is not defined %}None{% else %}{{ preboot_files }}{% endif %}
+
+    - debug:
+        msg: "Preboot-list: {{ preboot_list }} Preboot-files: {{ preboot_files }}"
+
+    - name: Set PTF test params
+      set_fact:
+        dut_mac: "{{ ansible_Ethernet0['macaddress'] }}"
+        vlan_ip_range: "{{ minigraph_vlan_interfaces[0]['subnet'] }}"
```

Comment:
That means you're going to check only first vlan interface?
What if you have many vlan interfaces?

Reply:
That was how the test was originally written. I just explicitly assigned them to vars since I was running into ansible errors while running the test in a loop for each preboot operation